inlong-sdk/transform-sdk/src/main/java/org/apache/inlong/sdk/transform/pojo/TransformConfig.java

```diff
@@ -35,16 +35,37 @@ public class TransformConfig {
     @JsonProperty("configuration")
     private Map<String, Object> configuration;
 
+    @JsonProperty("strictOrder")
+    private boolean strictOrder = true;
+
+    @JsonProperty("ignoreConfigError")
+    private boolean ignoreConfigError = true;
+
     @JsonCreator
     public TransformConfig(@JsonProperty("transformSql") String transformSql) {
-        this(transformSql, ImmutableMap.of());
+        this(transformSql, ImmutableMap.of(), true, true);
     }
 
     @JsonCreator
     public TransformConfig(@JsonProperty("transformSql") String transformSql,
             @JsonProperty("configuration") Map<String, Object> configuration) {
+        this(transformSql, configuration, true, true);
```

Comment:
false, false

Reply:
It is major scene that select field list is same order with sink field list.